### PR TITLE
Update linear.py

### PR DESCRIPTION
**Pull request by ymodak**
fixes #30092
https://github.com/tensorflow/tensorflow/issues/30092

### CODE REVIEW

**Comment by karmel** [comment]
I don't know that this will work as is-- @yhliang2018 @tanzhenyu , can you look at the attached issue, and suggest the correct fix here?

**Comment by yhliang2018** [comment]
@karmel This is to fix the doc string of optimizer in LinearClassifier, not the actual code. So it should be fine. I also comment on the issue thread.

**Comment by rthadur** [comment]
@karmel @yhliang2018 can you please suggest a reviewer who i can assign for this PR, thank you.

**Comment by yhliang2018** [comment]
@rthadur I can take it.

**Comment by rthadur** [comment]
> @rthadur I can take it.

@yhliang2018 thank you , but i don't see your name in reviewers list , can you please get added to that list.

**Comment by yhliang2018** [comment]
Which reviewers list should I be added? Could you share the link? Thanks!

**Comment by karmel** [comment]
Should work now. Thanks, all.

File: tensorflow_estimator/python/estimator/canned/linear.py [hunk 1]
```diff
@@ -796,7 +796,7 @@ class LinearClassifierV2(estimator.EstimatorV2):
   estimator = LinearClassifier(
       feature_columns=[categorical_column_a,
                        categorical_feature_a_x_categorical_feature_b],
-      optimizer=tf.train.FtrlOptimizer(
+      optimizer=tf.keras.optimizers.Ftrl(
```
Comment (yhliang2018) [inline tensorflow_estimator/python/estimator/canned/linear.py]: Could you also update all the other doc strings with tf.train.FtrlOptimizer? Like the one in line 808, and the ones in LinearEstimatorV2 and LinearRegressorV2?

Comment (tanzhenyu) [reply tensorflow_estimator/python/estimator/canned/linear.py]: +1.

Comment (ymodak) [reply tensorflow_estimator/python/estimator/canned/linear.py]: Made the changes downstream. Thanks!

**Pull request merged**